.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: scala
-script: sbt test scripted
+script: sbt ^validate
```

Comment:
You can use a build matrix here so that we can have better reports of which version of sbt failed:

```yaml
env:
  matrix:
    - TRAVIS_SBT_VERSION=0.13.16
    - TRAVIS_SBT_VERSION=1.0.0-RC3
script:
- sbt ^^$TRAVIS_SBT_VERSION test
- sbt ^^$TRAVIS_SBT_VERSION scripted
- sbt ^^$TRAVIS_SBT_VERSION publishLocal
```